### ISSUE TITLE
Fastidious Bloom filter requires more memory than indicated in the help

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/swarm#127]
```
-c, --ceiling positive integer
    when using the option --fastidious (-f), define swarm's maximum 
    memory footprint (in megabytes). [...] The value must be at least 3.
```
Testing with the smallest possible example triggering a fastidious grafting:
```sh
swarm -f -c 7 <(printf ">s1_3\nAA\n>s2_1\nCC\n")

Error: Insufficient memory remaining for Bloom filter

swarm -f -c 8 <(printf ">s1_3\nAA\n>s2_1\nCC\n")

OK
```
I'll modify the manpage. Maybe a test can be added to swarm when parsing the command line arguments to check that the input for `-c` is at least 8?

**Comment by frederic-mahe** [comment]
manpage modified (https://github.com/torognes/swarm/commit/83280ba19cb900a847791c1d6279a9cb76e5cb35)

**Comment by torognes** [comment]
Fixed in d2ade53.

**Comment by torognes** [comment]
Now fixed in zobrist branch too: f16a45f33a9f8b2bc6c2f3cd72388bd7520b2843

**Comment by frederic-mahe** [comment]
All tests regarding the ceiling value are now up to date (https://github.com/frederic-mahe/swarm-tests/commit/a08390b2fab014bfbea3e6a6c537151a6d454039).
